seed/models/sensors.py

```diff
@@ -6,21 +6,39 @@
 from seed.models import Property
 
 
-class Sensor(models.Model):
-    sensor_property = models.ForeignKey(
+class DataLogger(models.Model):
+    property = models.ForeignKey(
```

Comment:
nice. I like that we are attaching this onto the Property (similar to how sensor was on the property).